presto-docs/src/main/sphinx/connector/inmemory.rst

```diff
@@ -0,0 +1,55 @@
+==================
+InMemory Connector
+==================
+
+The InMemory connector stores all data and metadata in RAM on workers
+and both are discarded when Presto restarts.
```

Comment:
"Presto restarts" -> "Presto is restarted on all nodes in the cluster" --- correct?